cmd/gapir/cc/main.cpp

```diff
@@ -143,6 +143,46 @@ std::unique_ptr<Server> Setup(const char* uri, const char* authToken,
 
 #if TARGET_OS == GAPID_OS_ANDROID
 
+namespace {
+
+#define JNI_CALL_O(obj, name, sig, ...)                           \
```

Comment:
I would prefer to see this as a template, rather than a macro.

Pardon my bad pseudo-code and making up names of JNI api things:
```
template<typename... Args>
jobject jni_call_o(Env* env, clazz obj, const char* name, const char* sig, Args&&... args) {
   return env->CallObjectMethod(obj, env->GetMethodIf()......, std::forward<Args>(args)...);
}
```